src/com/shapestone/travelportal/Payment.java

```diff
@@ -0,0 +1,36 @@
+package com.shapestone.travelportal;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Payment {
+	public static void main(String... args) throws StreamReadException, DatabindException, IOException {
```

Comment:
Add driver program  for the all 3 stories, and give option to user for selecting menu

Reply:
Changes are done.Please review the code